Lib/asyncio/selector_events.py

```diff
@@ -1105,6 +1160,16 @@ def write_eof(self):
         if not self._buffer:
             self._sock.shutdown(socket.SHUT_WR)
 
+    def writelines(self, list_of_data):
```

Comment:
Nit: Maybe use the '/' notation to make this a positional-only argument?

Reply:
That would be backwards incompatible change.